package.json

```diff
@@ -125,6 +125,7 @@
     "react-router": "^2.3.0",
     "react-test-renderer": "^15.3.2",
     "redux": "^3.0.0",
+    "sitemap": "~1.8.2",
```

Comment:
Done. But the `yarn.lock` is not under version. So what is the point to run it locally ?